spec/models/solr_document_spec.rb

```diff
@@ -11,15 +11,6 @@
     end
   end
 
-  describe "to_param" do
```

Comment:
Nope, it shouldn't be -- this was from an earlier run through the code when I excised that completely. I'll add the test back.